modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJSPDLUtilCheck.testjsp

```diff
@@ -1,7 +1,4 @@
-/**
- * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
- * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
- */
+<%@ page import="com.liferay.portal.kernel.util.PortalUtil" %>
```

Comment:
@nicolas-sss, wouldn't be better create a dedicated commit for the test fixes?

Did you decide to include the test fixes to this commit to guarantee that it'll compile successfully?

Reply:
I prefer the fixes in the same commit to ensure the test's success independently of other commits.